test/rules/no-async-describe.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../../lib/rules/no-async-describe');
+const ruleTester = new RuleTester();
+
+ruleTester.run('no-async-describe', rule, {
+    valid: [
+        'describe()',
+        'describe(function () {})',
```

Comment:
The function should be the second argument of describe.

Reply:
Of course! :+1:

Fixed in 7c9b6dc71664d8756fca64ddbbf0bf7657f727e7